examples/cp-all-in-one/docker-compose.yml

```diff
@@ -91,3 +91,39 @@ services:
       CONFLUENT_METRICS_TOPIC_REPLICATION: 1
       PORT: 9021
 
+  ksql-server:
+    image: confluentinc/ksql-cli:4.1.0
+    container_name: ksql-server
+    depends_on:
+      - broker
+    ports:
+      - "8088:8088"
+    command: "bash -c -a 'echo Waiting for Kafka to be ready... && \
+                       sleep 1 && \
+                       ksql-server-start /tmp/ksql.properties 2>&1'"
+    volumes:
+      - $PWD/ksql.properties:/tmp/ksql.properties
+      - $PWD/monitoring-interceptors/monitoring-interceptors-4.1.0.jar:/usr/share/java/ksql-cli/monitoring-interceptors-4.1.0.jar
+      - $PWD/ui:/usr/ui
+      - $PWD/ksql-datagen-jars/ksql-examples-4.1.0.jar:/usr/share/java/ksql-cli/ksql-examples-4.1.0.jar
+      - $PWD/ksql-datagen-jars/kafka-connect-avro-converter-4.1.0.jar:/usr/share/java/ksql-cli/kafka-connect-avro-converter-4.1.0.jar
+      - $PWD/ksql-datagen-jars/generex-1.0.2.jar:/usr/share/java/ksql-cli/generex-1.0.2.jar
+      - $PWD/ksql-datagen-jars/automaton-1.11-8.jar:/usr/share/java/ksql-cli/automaton-1.11-8.jar
+    environment:
+      KSQL_CONFIG_DIR: "/etc/ksql"
+      KSQL_LOG4J_OPTS: "-Dlog4j.configuration=file:/etc/ksql/log4j-rolling.properties"
+
+  rest-proxy:
+    image: confluentinc/cp-kafka-rest:4.1.0
```

Comment:
any reason we want to hardcode the tags here in the examples?

Reply:
Hmmm, I'm hesitant to always make it latest because variables change sometimes.  But for this case with REST proxy, I think it's ok.  I changed to `latest`